.cursor/rules/unit-tests.mdc

```diff
@@ -0,0 +1,14 @@
+---
+description: 
+globs: 
+alwaysApply: true
+---
+When running unit tests, always include the flag `--watch=false` to prevent watch mode.
+
+To run unit tests for a specific package in the monorepo, use the following command:
+
+`yarn workspace <workspaceNameFromPackageJson> run test <pathToTest>`
+
+Note: The test runner may be either Vite or Jest, depending on the package. Check the `package.json` file in the relevant package to determine which test runner is used.
```

Comment:
Do we use jest any more?

Reply:
Nice! Looks like we only use Vitest now. Updated the instructions!